projects/goharbor/harbor/buildspecs/batch-build.yml

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 # This file was generated by running `make generate`. DO NOT EDIT
-
```

Comment:
We should keep this newline since the `yq` on builder-base generates it with newline, and removing it will cause the validation to fail.

Reply:
Updated